### ISSUE TITLE
Push nuget packages with source link support

### ISSUE THREAD

**Issue by Kees-Schouten** [elsa-workflows/elsa-core#2977]
It would be very nice if the Elsa packages are published to NuGet.org with source link support.

![elsa source link](https://user-images.githubusercontent.com/4984029/165958869-fc65d7cf-10d0-4fc3-adf2-ce0c27ab0c7c.png)

This way we can debug Elsa by just using the nuget packages instead of the complete source.

**Comment by sfmskywalker** [comment]
Hi @Kees-Schouten, is there any chance you could take a look at the [appveyor.yml](https://github.com/elsa-workflows/elsa-core/blob/master/appveyor.yml) file and see if something might be missing there in order to get the symbol files published?

I thought the following lines would suffice, but clearly this is not the case:

```yml
dotnet pack Elsa.sln --include-symbols /p:Version="$env:elsa_build_version"

deploy:
    # Deploy to NuGet on builds from tags only
    -   provider: NuGet
        name: NuGet
        api_key:
            secure: KKRDla0gYKmprH/5dN/dJag/n6uAdHU6Uq8CME4ONzz2AOsMPtzS+EI/MVuSNilD
        skip_symbols: false
        symbol_server: https://www.nuget.org/api/v2/symbolpackage # remove to push symbols to SymbolSource.org
        artifact: /.*(\.|\.s)nupkg/
        on:
            APPVEYOR_REPO_TAG: true
```

**Comment by jdevillard** [comment]
seems that we've to add also the package format for symbol 

`-p:SymbolPackageFormat=snupkg`


https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg#creating-a-symbol-package

**Comment by sfmskywalker** [comment]
Looks like this might be closed via #3181 - thanks @holidayfun !
